Automattic/woocommerce-payments
# correct case of `Id` for Transaction ID column label in transactions list screen

Fixes #9910 

#### Changes proposed in this Pull Request

Change column name `Transaction Id` to `Transaction ID` for consistency with how unique ID's are labelled across the UI. 

#### Testing instructions

1. Browse to Payments > Transactions
2. Click the ellipsis menu at the top right of the report table.
3. You should see `Transaction ID` instead of `Transaction Id` . Toggle that control on 

<img src="https://github.com/user-attachments/assets/a25afade-f68a-43c9-a5f9-d9e04eda1670" width="300px" />

4. In the report you should now see an additional column - `Transaction ID`

<img src="https://github.com/user-attachments/assets/0cd5a5de-3a5a-4bc8-a333-b8231d780c8e" width="300px" />


-------------------


**Post merge**
